docs/aad/MeetingConfigurableTab.md

```diff
@@ -0,0 +1,248 @@
+
+In this lab you will learn to:
+
+- Configure your Configurable Tab for use with Microsoft Teams meetings
+- Add your tab to the Microsoft Teams Meeting Pre-meeting stage
+- Add your tab to the Microsoft Teams Meeting Side Panel during a meeting
+- Present your tab in the Microsoft Teams Meeting Content Stage during a meeting
+- Use the Teams JS SDK to understand the current Microsoft Teams Frame Context
+- Use the Teams JS SDK to understand whether the tab is being shared to the Microsoft Teams Meeting Content Stage
+
+### Features
+
+- Microsoft Teams configurable tab to display a product category in during pre-meeting stage
+- Microsoft Teams configurable tab to display a product category in the Meeting Side Panel during a Teams meeting
+- Microosft Teams configurable tab to display a product category and be shared to the Meeting Content Stage
+
+### Exercise 1: Update your configurable tab for use with Meetings
+
+#### Step 1: Add context array to application manifest
+
+In your code editor, open the manifest/manifest.template.json file.
+
+Update the version number so it's greater than it was; for example if your manifest was version 1.4, make it 1.4.1 or 1.5.0. This is required in order for you to update the app in Teams.
+
+```json
+"version": "1.5.0"
+```
+
+> NOTE: Make each new version greater than the last so you can update the application in Teams!
+
+Now, immediately under the `scopes` array, in the your configurable tab object, add a new array for defining the `context`:
+
+```json
+"configurableTabs": [
+    {
+      "configurationUrl": "https://<HOST_NAME>/pages/tabConfig.html",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "teams",
+        "groupchat"
+      ],
+      "context": [
+        "meetingChatTab",
+        "meetingDetailsTab",
+        "meetingSidePanel",
+        "meetingStage"
+      ]
+    }
+  ]
+```
+
+#### Step 2: Rebuild your application package
+
+Open a command line tool in your working folder and execute
+
+```sh
+npm run package
+```
+
+This will generate a new `manifest.json` file and a new application package (northwind.zip).
+
+#### Step 3: Upload the app package
+
+In the Teams web or desktop UI, click "Apps" in the sidebar 1️⃣, then "Manage your apps" 2️⃣. At this point you have three choices:
+
+* Upload a custom app (upload the app for yourself or a specific team or group chat) - this only appears if you have enabled "Upload custom apps" in your setup policy; this was a step in the previous lab
+* Upload an app to your org's app catalog (upload the app for use within your organization) - this only appears if you are a tenant administrator
+* Submit an app to your org (initiate a workflow asking a tenant administrator to install your app) - this appears for everyone
+
+In this case, choose the first option 3️⃣.
```

Comment:
From @rabwill 

If I may suggest for uploading the package if we can choose option 2 (org app catalog) instead of 1 or 3 (submit an app to org).
option 1 will ask user to add the app, which is not captured in the lab.
option 3 will kick off an approval flow with admin for approval, which is not captured in the lab.

Also the instruction is contradicting In this case, choose the first option 3️⃣.

Reply:
This step was copied verbatim from the Configurable Tab lab, so bringing @BobGerman into this.

I understand where the confusion comes from with the final instruction, however my understanding is that the 3️⃣ refers to the third step shown in the image below, which highlights to the student that they should select the `Upload a custom app` option which is the first in the list.

In either case, choosing option one or option two would work for this lab, however I'm wondering whether it is relevant to describe all three options to the student here and instead just direct them to the option that they need to select to get to the next step in the lab. 

In the `Upload an app` dialog shown in Teams, each available option has descriptive text and a link to learn more which the student can use to dive deeper if they wish.

Based on the above, I'd like to suggest that we make this step more concise to keep the student on track, with the image following the below text. Thoughts?

```suggestion
In the Teams web or desktop UI, click "Apps" in the sidebar 1️⃣, then "Manage your apps" 2️⃣, select `Upload a custom app` 3️⃣ , navigate to the manifest directory, select `Northwind.zip` file and upload it. 

Although the application is already installed, you are providing a newer version so it will update the application.
```